### DISCUSSION TITLE
Maxilla Reconstruction - Proximal fibula segment switches with distal segment

### DISCUSSION THREAD

**Discussion by qferreira-1** [lassoan/SlicerBoneReconstructionPlanner#104] [Q&A]
Looking for some feedback how to avoid the bone segments switching places. I am currently on 5.03 as this case was originally created on that version. (I do have 5.4 installed if needed). But essentially this happens for a few cases where we need to shift a segment, but then the fibula segments completely switch spots. Photo for reference.
![image](https://github.com/SlicerIGT/SlicerBoneReconstructionPlanner/assets/122468415/a018af5a-0ee4-475a-93db-5f330803058d)
 So the pink segment is proximal, and needs to remain as such. However when lowering he plane, the segment switches (see photo).
![image](https://github.com/SlicerIGT/SlicerBoneReconstructionPlanner/assets/122468415/38e36238-9412-43a3-a0f0-25749ca484c0)
So from the fib guide, you can see the angles of the original cuts vs the new lowered plane cut. The distal and proximal segments have switched. 
The fibula was wrapped, and the maxilla is solid with no voids/empty voxels. 

Any help would be greatly appreciated

**Comment by qferreira-1** [comment]
Here is a photo of the maxilla with the issue.
![image](https://github.com/SlicerIGT/SlicerBoneReconstructionPlanner/assets/122468415/5331694e-9d9d-4003-9cf9-9b0dacd18deb)
The plane was only transformed lower, and was adjusted to this.

**Comment by mauigna06** [comment]
Hi @qferreira-1 

Thanks for reporting this.
I would need to be able to replicate this bevahior. You can send me an Slicer scene of a surgical plan where this happens. A video or detailed instructions could do.
I have some deadlines at work but I'll try to solve this as fast as I can.

Thank you,
Mauro

**Reply by qferreira-1** [reply]
No rush, I have found ways to navigate around this with manipulation of the planes and fibula line. I will post a video shortly of the scene and my steps to get to where I was. If I can anonymize the data quickly, I will send the scene as well.
